bin/build-distro

```diff
@@ -53,3 +53,8 @@ fi
 
 sudo rm -Rf rootfs
 echo "${SERIAL}" > serial
+
+# Calculate checksums for all files within a directory.
+for file in *; do
+    sha256sum "${file}" >> SHA265SUMS
+done
```

Comment:
The only reason I see to use a `for` loop is if you want to avoid any existing `SHA256SUMS` file i the current directory. If that's your goal, I'd add this:
```
for file in *; do
  [ "${file}" = "SHA256SUMS" ] && continue
  ...
done
```

Otherwise, a simple `sha256sum * > SHA265SUMS` would do.

Reply:
Nice, didn't know that will work. Thanks